README.md

```diff
@@ -7,105 +7,18 @@ Pre-release version of Coding Coach.
 
 ## Looking for a mentor?
 
-Just go to https://mentors.codingcoach.io/ and find her / him / them.
+Just go to https://mentors.codingcoach.io/, register and find her / him / them.
```

Comment:
I need an English native here! It is possible to use a more inclusive pronoun instead of her/him? `They` maybe?

Reply:
Perhaps use `a mentor` instead of a pronoun at all?

Reply:
Done :) Thanks